websites/W/wumpus.store/presence.ts

```diff
@@ -0,0 +1,59 @@
+const presence = new Presence({
+		clientId: "998532462755979275",
+	}),
+	browsingUnix = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/KaW8pr2.png",
+			details: "Viewing Home Page",
+			startTimestamp: browsingUnix,
+		},
+		{ pathname, origin } = document.location;
```

Comment:
Since we replace origin with href:
```suggestion
		{ pathname, href} = document.location;
```